packages/grid/data-grid/rollup.config.js

```diff
@@ -44,14 +44,11 @@ export default [
     output: [{ file: 'dist/data-grid.d.ts', format: 'es' }],
     plugins: [
       dts(),
-      command(
-        `cat ../x-grid-modules/dist/x-grid-modules.d.ts ./dist/data-grid.d.ts > ./dist/data-grid.all.d.ts`,
```

Comment:
Should we update

https://github.com/mui-org/material-ui-x/blob/009577f1de980aeea897de34a42d721e6b668734/packages/grid/data-grid/package.json#L8

too?